deploy/rbd/deploy.md

```diff
@@ -0,0 +1 @@
+docs/rbd/deploy.md
```

Comment:
accessing symlink failed for me, can you please cross check this and see if you are able to access the file if you open it from deploy folder?

Reply:
yes, shows error while opening, will check on that

Reply:
it works now, please do review it again